README.rst

```diff
@@ -0,0 +1,166 @@
+========================================================
+ The Community Earth System Model version 2.0 (CESM2.0)
+========================================================
+
+See the CESM web site for documentation and information:
+http://www2.cesm.ucar.edu
+
+.. sectnum::
+
+.. contents::
+
+Obtaining the full model code and associated scripting infrastructure
+=====================================================================
+
+CESM2.0 is now released via github. You will need some familiarity with git in order
+to modify the code and commit these changes. However, to simply checkout and run the
+code, no git knowledge is required other than what is documented in the following steps.
+
+To obtain the CESM2.0 code you need to do the following:
+
+#. Clone the repository. ::
+
+      > git clone https://github.com/escomp/cesm.git my_cesm_sandbox
```

Comment:
Is there a minimum git version that needs to installed locally?

Reply:
No idea. @bandre-ucar do you have any idea?

Reply:
For manage_externals... Probably. We are using system calls to the command line options and potentially screen output as an API. This as generally considered Not A Good Idea (TM) because they aren't guaranteed to be a stable. It why shell scripts using non-posix interfaces are so hard to port and maintain...

I have only tested with:
* git - 2.10, 2.15.
* svn - 1.8.10, 1.9.4

I don't expect it to work with git 1.x

Reply:
@bertinia and @bandre-ucar So do you feel we should add a comment at the top of the README to the effect of, "You must have a git version 2.x client for this to work correctly"? (I'm not too concerned about svn, because the basic svn commands that I expect the tool uses have not changed in a long time.)

Reply:
It looks like the default git on hobart is 1.8. I cloned escomp and ran checkout_externals and it worked. I think we are fine for now, but we should be aware of this going forward.